src/Extension/MemberExtension.php

```diff
@@ -39,4 +44,34 @@ public function getDefaultRegisteredMethod()
     {
         return $this->owner->RegisteredMFAMethods()->byId($this->owner->DefaultRegisteredMethodID);
     }
+
+    public function updateCMSFields(FieldList $fields)
+    {
+        Requirements::javascript('silverstripe/mfa: client/dist/js/bundle-cms.js');
+        Requirements::css('silverstripe/mfa: client/dist/styles/bundle-cms.css');
+
+        $fields->removeByName('DefaultRegisteredMethodID');
+        $fields->removeByName('HasSkippedMFARegistration');
+        $fields->removeByName('RegisteredMFAMethods');
```

Comment:
You could pass an array of strings to minimise lines

Reply:
Thanks for the tip!